internal/character/danhengimbibitorlunae/ult.go

```diff
@@ -0,0 +1,72 @@
+package danhengimbibitorlunae
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	Point            = "danhengimbibitorlunae-point"
+	Ult   key.Attack = "danhengimbibitorlunae-ult"
+	E2    key.Reason = "danhengimbibitorlunae-e2"
+)
+
+var ultHits = []float64{0.3, 0.3, 0.4}
+
+func init() {
+	modifier.Register(Point, modifier.Config{
+		StatusType:        model.StatusType_STATUS_BUFF,
```

Comment:
Squama Sacrosancta in datamine is  "StatusType": "Other", i think in this case the `StatusType:` need to be removed.
```suggestion
```